### ISSUE TITLE
Possible integer overflow in InventoryMenu::onTakeStuff

### ISSUE THREAD

**Issue by Try** [Try/OpenGothic#45]
Miss this minor issue on code review: 
```c++
void InventoryMenu::onTakeStuff() { 
  uint8_t itemCount = 0;
  auto& page = activePage();
  auto& sel = activePageSel();
  if(sel.sel >= page.size())
    return;
  auto& r = page[sel.sel];
  if(lootMode==LootMode::Normal) {
    ++takeCount;
    itemCount = pow(10,takeCount / 10);
    if(r.count() <= itemCount) {
      itemCount = r.count();   // <<----- HERE: uint8_t = size_t
      takeCount = 0;
    }
  }
```
While it's hard to meet this condition in real game, still should be fixed

**Comment by Try** [comment]
@da3m0nsec da3m0nsec, can you take a look on this issue?

**Comment by da3m0nsec** [comment]
Fixed on #46
